tests/phpunit/elementor/core/breakpoints/test-manager.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Breakpoints;
+
+use Elementor\Core\Breakpoints\Breakpoint;
+use Elementor\Core\Breakpoints\Manager as Breakpoints_Manager;
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+
+class Test_Manager extends Elementor_Test_Base {
+
+	/**
+	 * Test Get Name
+	 *
+	 * Test the Breakpoints_Manager::get_name() method.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_name() {
+		$this->assertEquals( 'breakpoints', Plugin::$instance->breakpoints->get_name() );
+	}
+
+	/**
+	 * Test Get Breakpoints
+	 *
+	 * Test the Breakpoints_Manager::get_breakpoints() method.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_breakpoints() {
+		$breakpoints = Plugin::$instance->breakpoints->get_breakpoints();
+		$all_breakpoint_names = array_keys( Breakpoints_Manager::get_default_config() );
+
+		foreach ( $breakpoints as $breakpoint_name => $breakpoint ) {
+			/** @var Breakpoint $breakpoint */
+			// Check that the items in the breakpoints array are actual breakpoints.
+			$this->assertTrue( $breakpoint instanceof Breakpoint );
+
+			// Check that the generated breakpoint is one of the breakpoints in the default config array.
+			$this->assertContains( $breakpoint_name, $all_breakpoint_names );
+		}
```

Comment:
Instead of that compare it using `array_intersec_key`

Reply:
Not sure I understand how you mean.

Reply:
Use `array_diff_key`

Reply:
Done.